dashboard/app/dsl/evaluation_multi_dsl.rb

```diff
@@ -0,0 +1,10 @@
+class EvaluationMultiDSL < MultiDSL
+  def answer(text, weight: weight = 1, learning_module: learning_module = nil)
+    answer = {text: text}
+    raise 'Weight must be an integer' unless weight.is_a?(Integer)
+    answer[:weight] = weight || 1
+    raise "Unknown learning module #{learning_module}" unless learning_module.nil? || Plc::LearningModule.exists?(name: learning_module)
```

Comment:
I don't think `LearningModule` names are guaranteed to be globally unique?  Also should this be referencing `Stage`?